Segments/Axe Glacier.mapproj

```diff
@@ -104950,9 +104950,9 @@
         new LootPackEntry(true, TestTreasureBottles,    20),
         new LootPackEntry(true, TestTreasureTreasure,   0.63),
             
-            new LootPackEntry(true, (from, container) => new PearDiamond(1500u),    100, gemsPriceMutator),
-            new LootPackEntry(true, (from, container) => new GriffinSkull(),        100, gemsPriceMutator),
-            new LootPackEntry(true, (from, container) => new SilverDagger(),        100, gemsPriceMutator)
+            new LootPackEntry(true, (from, container) => new PearDiamond(3000u),    100, gemsPriceMutator),
+            new LootPackEntry(true, (from, container) => new DrakeScaleArmor(),        100, gemsPriceMutator),
```

Comment:
These should only be available when tanned? May need to remove the drake scale armor.